whatsappHandler.go

```diff
@@ -177,14 +177,21 @@ func (wh *waHandler) HandleTextMessage(message whatsapp.TextMessage) {
 			}
 		}
 
-		reply, parserErr := nemoParser(BuildCommands[index].Message, Sessions)
-		if parserErr != nil {
-			log.Println(parserErr.Error())
+		txt, imgs, err := nemoParser(BuildCommands[index].Message, Sessions)
+		if err != nil {
+			log.Println(err.Error())
 			return
 		}
 
-		if reply != "timeout" {
-			go sendMessage(wh.c, reply, message.Info.RemoteJid)
+		// non-nil txt indicates is it a text message
+		if txt != nil {
```

Comment:
does not rewrite `reply` even if message is parsed successfully, I think the next process has separate logic for determining `reply` status. should not affect your current code but this might be breaking.